fabfile.py

```diff
@@ -44,7 +49,24 @@ def create_site(name):
 
 @task
 def build():
-    local("docker build .")
+    """BUILD the docker image"""
+    timestamp = datetime.datetime.now().strftime("%Y%m%d%H%M%S")
+    print(timestamp)
+    tag = "docker.io/caktus/opendebates:%s" % timestamp
+    local("docker build . --tag=%s" % tag)
+
+
+@task
+def publish():
+    """PUBLISH the most recent docker image"""
+    # Find the most recent
+    tag = local(
+        "docker images caktus/opendebates --format '{{.Tag}}' | head -1", capture=True
+    )
+    if tag == "":
+        abort("No docker image found, have you run 'fab build' yet?")
+    print(tag)
+    local("docker push caktus/opendebates:%s" % tag)
```

Comment:
At some point, it could be nice to build/push/deploy all from Travis (again, probably as a separate task).